docs/usage.md

```diff
@@ -133,7 +133,7 @@ nf-core/raredisease consists of several tools used for various purposes. For con
 5. Variant calling - Structural variants (SV) (Tiddit & Manta)
 6. SNV annotation & ranking (rohcall, vcfanno, ensembl VEP, GENMOD)
 7. SV annotation & ranking (SVDB query, ensembl VEP, GENMOD)
-8. Mitochondrial analysis
+8. Mitochondrial annotation
 
```

Comment:
Mitochondrial annotation is 8 here, but was 9 before (line 23) and is 9 later on (line 245), check if it's correct